optaplanner-core/src/test/java/org/optaplanner/core/api/score/stream/ConstraintCollectorsTest.java

```diff
@@ -194,7 +194,7 @@ public void sum() {
         // Add second value, we have two now.
         int secondValue = 1;
         Runnable secondRetractor = accumulate(collector, container, secondValue);
-        assertResult(collector, container,firstValue + secondValue);
+        assertResult(collector, container, firstValue + secondValue);
```

Comment:
In other tests, I have the tendency:
1) not to make them too smart, by actually doing math. For example:
int a = Integer.MAX +1;
int b = Integer.MAX +1;
assertEquals(calculator.sum(a, b), a + b); // green if buggy

2) Use numbers such as 1000, 200, 30, 4, 0.5, ... etc that are very easy to sum up.

So I'd argue that we can improve readability and safety by doing this:
int firstValue = 100;
int secondValue = 20;
assertResult(collector, container, 120);